include/TPP/Dialect/Tpp/TppUtils.h

```diff
@@ -43,7 +43,7 @@ bool hasTppMark(linalg::LinalgOp linalgOp);
 bool isMarkedWithTpp(linalg::LinalgOp linalgOp, const std::string &target);
 
 // Returns true if the linalg operation has a Matmul region.
-bool hasMatmulBody(linalg::LinalgOp linalgOp);
+bool hasFMABody(linalg::LinalgOp linalgOp);
```

Comment:
Personally I don't like this rename. It gives the wrong impression that we are matching for FMA ops, which is not the case.

Reply:
your not matching MatMul either... just looking for Mul/Add is not a Matmul.

Reply:
`hasAddMulBody`?

Reply:
`hasMulAddBody`? To get the order right? :smile:

Reply:
Oh, and don't forget to update the comment, too.